providers/instantly/delete.go

```diff
@@ -0,0 +1,39 @@
+package instantly
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+// Delete removes object. As of now only removal of Tags are allowed.
```

Comment:
```suggestion
// Delete removes object. As of now only removal of Tags are allowed because
// deletion of other object types require a request payload to be added, c.Client.Delete does not yet support this.
```

Reply:
Slight difference: it uses DELETE with payload. Not POST.